### ISSUE TITLE
Problems with dynamic fields

### ISSUE THREAD

**Issue by lucafagherazzi** [elementor/elementor#13409]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest development version of Elementor.

## Description

I have encountered problems with dynamic fields: on some there are no tools while on the links nothing can be selected.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro) with development versions.
- [x] This bug happens with a vanilla WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

![01](https://user-images.githubusercontent.com/40598133/103173268-15818b80-485a-11eb-8f68-4502e76cda59.jpg)
![02](https://user-images.githubusercontent.com/40598133/103173270-1b776c80-485a-11eb-972b-768966497cf8.jpg)
![03](https://user-images.githubusercontent.com/40598133/103173271-1f0af380-485a-11eb-94ff-8361bcc9f8e3.jpg)

**Comment by shilo-ey** [comment]
Hi @lucafagherazzi 

Can you please specify which widget are you using?

What are the steps to reproduce? 
Does this action is working on a stable version and not working on the dev-version?

Thanks

**Comment by DorShahar** [comment]
Merged issue

I merged this issue with  #13355 , which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.
Thanks for your feedback!
